docs/user/migration/external-name-alias.md

```diff
@@ -2,11 +2,10 @@
 
 ## Context
 
-Istio with 1.21 version has changed the behavior of the Service of type ExternalName.
-With the change, the Service of type ExternalName is treated as an alias of the Service that it points to.
-Istio introduced this change to align the way it handles the Service of type ExternalName with the Kubernetes behavior.
-Since it caused some issues for our users, we introduced a new annotation, **disable-external-name-alias**, to disable this change.
-However, use of this parameter is not recommended and if you are using it, please update your configuration according to the following points.
+To align with Kubernetes behavior, Istio with 1.21 version changed the behavior of the Service of type ExternalName.
```

Comment:
```suggestion
To align with Kubernetes behavior, Istio 1.21 changed the behavior of the Service of type ExternalName.
```
Can we say it like that maybe? For me, it's shorter and easier to understand. What do you think? I'm also wondering if we need to capitalize "Service"? When I read the Isti 1.21 upgrade note, it seems like it can be any service. 
```suggestion
To align with Kubernetes behavior, Istio 1.21 changed the behavior of a service of type ExternalName.
```
or is the Service a resource?

Reply:
Yes, it's Kubernetes Service of type ExternalName. See Kubernetes docs: https://kubernetes.io/docs/concepts/services-networking/service/#externalname
Istio is not consistent with Kubernetes guidelines and they do not capitalize the names of Kubernetes resources' kinds (like Pods, Services, Doployments, etc.) and we do